agent/agent.go

```diff
@@ -1056,8 +1063,14 @@ func (a *agent) handleReconnectingPTY(ctx context.Context, logger slog.Logger, m
 					break
 				}
 				rpty.activeConnsMutex.Lock()
-				for _, conn := range rpty.activeConns {
-					_, _ = conn.Write(part)
+				for cid, conn := range rpty.activeConns {
+					_, err = conn.Write(part)
+					logger.Debug(ctx,
+						"wrote to active conn",
+						slog.F("other_conn_id", cid),
+						slog.F("data", part),
+						slog.Error(err),
+					)
```

Comment:
Let's add bytes written, just in case:

```suggestion
					n, err := conn.Write(part)
					logger.Debug(ctx,
						"wrote to active conn",
						slog.F("other_conn_id", cid),
						slog.F("data", part),
						slog.F("bytes_written", n),
						slog.Error(err),
					)
```

That said, maybe this should be conditionally enabled. Logging here could lead to increased lag/stutter in the web terminal. Agents are nearly always run in verbose mode which logs this.

Reply:
We need to be able to log at a level that isn't enabled by default in production.  Debug is the lowest level, and so that's what we're logging here.

We need to stop running `Debug` logging by default in production, otherwise what even is the point of having multiple levels?

I'm willing to drop this log as part of this PR, but I'll open another PR shortly that changes our default production logs to `Info`.

Reply:
I'm going to change this to log `if err != nil` and drop logging the `part` since it could contain sensitive data.